src/utils/waitForHttp.ts

```diff
@@ -1,7 +1,8 @@
 import type { IIoK8sApiCoreV1Container } from "kubernetes-models/_definitions/IoK8sApiCoreV1Container";
 
-// renovate: datasource=docker depName=registry.gitlab.factory.social.gouv.fr/socialgouv/docker/wait-for-http versioning=5.0.1
-const SOCIALGOUV_DOCKER_WAIT_FOR_HTTP_VERSION = "5.0.1";
+// renovate: datasource=docker depName=ghcr.io/socialgouv/docker/wait-for-http versioning=6.0.1
+const SOCIALGOUV_DOCKER_IMAGE = "ghcr.io/socialgouv/docker/wait-for-http";
+const SOCIALGOUV_DOCKER_VERSION = "6.0.1";
```

Comment:
The renovate commit should be next to the version variable 

```suggestion
const SOCIALGOUV_DOCKER_IMAGE = "ghcr.io/socialgouv/docker/wait-for-http";
// renovate: datasource=docker depName=ghcr.io/socialgouv/docker/wait-for-http versioning=6.0.1
const SOCIALGOUV_DOCKER_VERSION = "6.0.1";
```